core/src/main/scala/org/apache/mahout/math/algorithms/preprocessing/AsFactor.scala

```diff
@@ -29,8 +29,18 @@ import org.apache.mahout.math.scalabindings._
 import org.apache.mahout.math.scalabindings.RLikeOps._
 import MahoutCollections._
 
+/**
+  * AsFactor is a preprocessor that turns a vector of integers into a vector of one-hot encoded
```

Comment:
I'd change this to "matrix" instead of "vector of integers"

Reply:
fair- i think it's a matrix of row vectors of integers